redis.cfg

```diff
@@ -33,12 +33,12 @@ redis-logfile = ${buildout:directory}/var/log/redis.log
 recipe = collective.recipe.scriptgen
 cmd = ${redis:location}/src/redis-server
 arguments =
-  ${buildout:directory}/redis.conf
+  ${buildout:bin-directory}/redis.conf
 
 [redis-conf]
 recipe = collective.recipe.template
-input = redis.conf.tpl
-output = redis.conf
+input = buildout-templates/redis.conf.tpl
+output = ${buildout:bin-directory}/redis.conf
```

Comment:
@mattss @mgrbyte I like the idea of having a nice tidy project root with just bootstrap and buildout configs, then mirroring the [Linux Filesystem Hierarchy Standard](http://www.pathname.com/fhs/). Buildout already creates a bin/ and most things create a var/ directory so I'd like to see a etc/ where all the configs go.

I know this goes against the grain (if there is such a thing) of buildouts, but at the very least can we put it in parts/redis/etc/ like supervisor does (for example - parts/supervisor/etc/supervisord.conf) ?